modules/addons/openprovider/Controllers/System/SynchroniseController.php

```diff
@@ -118,11 +119,15 @@ protected function getOpenproviderScheduledTransfers($offset = 0, $limit = 1000)
 
         $found_domains = $this->openProvider->api->searchDomain($filters)['results'];
 
-        if(!is_null($found_domains) && count($found_domains) == 1000) {
-            $new_offset = $offset + $limit;
-            $found_domains = $found_domains + $this->getOpenproviderScheduledTransfers($new_offset, $limit);
+        if(!is_array($found_domains)){
+            return $found_domains;
```

Comment:
just to confirm,
Better to put some comments what could be the results apart from an array

Reply:
Comments have been added